moto/config/models.py

```diff
@@ -68,6 +69,29 @@
     "AWS::IAM::Policy": policy_config_query,
 }
 
+CONFIG_REGIONS = [
```

Comment:
Why is this required? Regions for the service should be part of botocore. If you do need to get a list of all active regions, you can do something along the lines of:

```
import boto3
...
CONFIG_REGIONS = boto3.Session().get_available_regions("config")
```

This looks at the botocore JSON model and does not make a call out to AWS.

Reply:
That's a handy trick! I had no idea you could do that. Replaced!